openapi3filter/validate.go

```diff
@@ -0,0 +1,11 @@
+package openapi3filter
+
+import (
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+// RegisterArrayUniqueItemsChecker is used to register a customized function
+// used to check if JSON array have unique items.
+func RegisterArrayUniqueItemsChecker(fn func(items []interface{}) bool) {
```

Comment:
Why repeat this here? Isn't the definition in package `openapi3` enough?

Reply:
From my point of view this function is used for validation， here is openapi3filter，namespace，so I think it’s better register function from openapi3filter. what you think?

Reply:
I prefer to not duplicate names + all schema validation code is already within `openapi3`. I'd rather keep it there & not defining this here.

You've documented some of this in the main README but maybe you should also mention this in other places that make sense to you? Maybe `openapi3filter` docs & `openapi3` validation docs?

Reply:
Do you mean add new README file under openapi3 folder?

Reply:
I was thinking of possibly adding documentation to this line: https://github.com/getkin/kin-openapi/blob/faf7556f33b5c39d08d7a54efbad5f13f319fa49/openapi3filter/validate_response.go#L14
Such as:
> ValidateResponse returns a non-nil error if the ResponseValidationInput does not match the OpenAPIv3 spec.
Note: one can tune the behavior of `uniqueItems: true` verification by registering a custom function with openapi3.RegisterArrayUniqueItemsChecker ...